lib/bitly_api_credentials.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
+
+# Copyright 2018 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This module manages access to our API keys for the bitly API
+
+import ConfigParser
+import warnings
+
+KEY_CFG_PATH = 'bitly_api_key.cfg'
+KEY_CFG_HEADER = 'Production'
+
+def get_client_key(file_path, header):
+	config = ConfigParser.SafeConfigParser()
```

Comment:
It looks like the other files are using spaces instead of tabs? Should probably have a consistent style.

Reply:
Done